lib/eb_deployer/resource_stacks.rb

```diff
@@ -1,15 +1,19 @@
 module EbDeployer
   class ResourceStacks
-    def initialize(resources, cf_driver, skip_provision=false)
+    def initialize(resources, cf_driver, skip_provision=false, tags)
```

Comment:
You really can't make `skip_provision` have a default when `tags` won't. This is a bug because if you decide to give only 3 params, it will fill `skip_default` and throw an error because you didn't supply `tags`. Better to make them all required rather than give a default to `tags` as well.